stable/openvpn/templates/config-openvpn.yaml

```diff
@@ -16,7 +16,7 @@ data:
     then
       echo "found existing certs - reusing"
       {{- if .Values.openvpn.useCrl }}
-      if [ ! -e ${EASY_RSA_LOC}/pki/crl.pem ]
+      if [ ! -e ${EASY_RSA_LOC}/crl.pem ]
```

Comment:
So is mounting a crl currently broken on master?

Reply:
Not sure if you’ve seen the cp command on line 23

Reply:
I think so - the cp command is making it work afaik, because if `${EASY_RSA_LOC}/crl.pem` exists then the code in the if statement has already run or we're using a keystore secret

Reply:
also, just noticed that the readonly keystore secret is enforced by the chart (https://github.com/helm/charts/blob/master/stable/openvpn/templates/openvpn-deployment.yaml#L72), so could change that and `chmod nobody:group crl.pem` instead? Not sure if you have a preference?

Personally I feel keeping the readonly mount is better.

Reply:
Changing the permissions of a file that’s been volume-mounted isn’t something I’ve ever tried, does it even work? If so, yes - I agree it’s better to keep the readonly mount. Another option is to mount the file as something like `crl-mounted.pem`, then `cp crl-mounted.pem crl.pem` and then `chmod 744 crl.pem`.

Reply:
yeah it works as long as the filesystem can be written to - a cp will only work but only if it's copied out of the mount point (`/etc/openvpn/certs`) and so `crl.pem` would live outside of the certs directory which feels a bit odd.

So I reckon either keeping this PR as is, or changing the mount point to `/etc/openvpn/certs/pki`, doing `cp /etc/openvpn/certs/pki/crl.pem /etc/openvpn/certs/crl.pem` and then `chown nobody:nogroup /etc/openvpn/certs/crl.pem`?

Reply:
Changing the mountpoint is fine, as long as it doesn’t affect mounting a server conf file or anything like that. I also think it’s ok for the crl.pem to live outside the certs directory and on the level of the config file, since it’s not really a cert itself anyway.

Reply:
cool, have updated the mount point and chowned to nobody:nogroup everywhere - think that works better

Reply:
Had to revert that changes because easyrsa has issues with the pki directory existing already - have gone back to the simpler implementation

Reply:
Had to revert that changes because easyrsa has issues with the pki directory existing already - have gone back to the simpler change